apps/test/util/runTests.js

```diff
@@ -0,0 +1,68 @@
+var which = require('npm-which')(__dirname).sync;
+var mochify = require('mochify');
+var http = require('http');
+
+// TODO (brent) When we move to npm/gulp, we will want to take care of this
+// logic in our gulpfile instead
+// Right now we manually create a symbolic link to @cdo/apps
+var fs = require('fs');
+var exec = require('child_process').exec;
+var target = __dirname + '/../../src/';
+var nodePath = __dirname + '/../../node_modules/@cdo/apps';
+var command = '';
+if (!fs.existsSync(target)) {
+  command = 'ln -s ' + target + ' ' + nodePath;
+}
```

Comment:
Thanks for the context. This does seem to work for now, but may fall apart once we gulpify cdo/shared if we need to access files between the two packages.  A good area to keep an eye on.